src/utils/hash.c

```diff
@@ -23,15 +20,15 @@ uint32_t md_hash(const char *msg, size_t length) {
 
   // Loop over the message 32-bits at-a-time
   while (length >= 4) {
-    os_memcpy(&block, msg, sizeof(uint32_t));
+    memcpy(&block, msg, sizeof(uint32_t));
```

Comment:
Why `os_memcpy` renamed to `memcpy`?

Reply:
It's to remove a dependency on `"allocs.h"`

It's just a macro to `memcpy` anyway, so I don't really see a point in using `os_memcpy`, see https://github.com/nqminds/edgesec/blob/84b412ff40720566c43134a2a23eb8e44c68aa50/src/utils/allocs.h#L98-L100

The only time I can see it being useful is if you're compiling to a platform that doesn't have a C standard library, but pretty much every platform has a C standard library.